cmd/processor/app/processor.go

```diff
@@ -193,7 +193,7 @@ func NewProcessor(configurationPath string, platformConfigurationPath string) (*
 	// if default http trigger creation is disabled and there is no any other http trigger we need to start internal
 	// healthcheck service
 	if (processorConfiguration.Spec.DisableDefaultHTTPTrigger == nil && platformConfiguration.DisableDefaultHTTPTrigger ||
-		*processorConfiguration.Spec.DisableDefaultHTTPTrigger) &&
+		processorConfiguration.Spec.DisableDefaultHTTPTrigger != nil && *processorConfiguration.Spec.DisableDefaultHTTPTrigger) &&
```

Comment:
Maybe just for readibilty, add more parenthesis:
```golang
	if ((processorConfiguration.Spec.DisableDefaultHTTPTrigger == nil && platformConfiguration.DisableDefaultHTTPTrigger) ||
		(processorConfiguration.Spec.DisableDefaultHTTPTrigger != nil && *processorConfiguration.Spec.DisableDefaultHTTPTrigger)) &&
```

Reply:
I'd say that those two are on the same "logic level", so maybe leave as is?